fix: warn if Terragrunt dependency is outside project. (#1983)
terramate-io/terramate: CHANGELOG.md

### Fixed

- Fix the command-line parsing of `run` and `script run` which were not failing from unknown flags.

> - Panic in the Terragrunt integration when the project had modules with dependency paths outside the current Terramate project.

Reviewer: ```suggestion
- Fix crash in the Terragrunt integration when the project had modules with dependency paths outside the current Terramate project.
```

>   - Now Terramate throw a warning for such configurations.

Reviewer: ```suggestion
  - A warning will be shown for such configurations.
```
